docs/site/src/content/docs/content/assets-views.mdx

```diff
@@ -259,6 +259,73 @@ Within a template, the `_index_` string can be used to substitute the array-inde
 }
 ```
 
+### Template ordering
+
+When writing template content, the position of the template in relation to non-template content is not guaranteed when it gets transformed into its final JSON output. Because of this, templates support a `placement` property that specifies whether the template should expand before or after pre-existing content.
+
+`placement?: "prepend" | "append"`
+
+Placement is supported on multiple templates, and will expand accordingly in relation to other templates and non-template data.
+
+The expansion logic will respect the following order:
+
+- prepend
+- existing data
+- append
+
+**_Example_**
+
+The following content will:
+
+- Place the expanded static template first, since it has `placement: prepend`
+- Then display content from the `values` array
+- Finally place the expanded dynamic template last, since it has `placement: append`
+
+```json
+{
+  "asset": {
+    "id": "top-level",
+    "type": "collection",
+    "values": [
+      {
+        "asset": {
+          "id": "existing-name",
+          "type": "text",
+          "value": "Something hard-coded"
+        }
+      }
+    ],
+    "template": [
+      {
+        "data": "list.of.names",
+        "output": "values",
+        "placement": "append",
+        "dynamic": true,
+        "value": {
+          "asset": {
+            "id": "dynamic-_index_",
+            "type": "text",
+            "value": "{{list.of.names._index_}}"
+          }
+        }
+      },
+      {
+        "data": "list.of.names",
+        "output": "values",
+        "placement": "prepend",
+        "value": {
+          "asset": {
+            "id": "static-_index_",
+            "type": "text",
+            "value": "{{list.of.names._index_}}"
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
```

Comment:
Even though we support `placement` now, we should document the fallback lexicographical effects on template expansion

Reply:
That would also close #540